ci/travis_before_install

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh -b -p $PWD/miniconda -f
+
+export PATH=$PWD/miniconda/bin:$PATH
+
+command -v conda || { echo conda not found ; exit 1; }
+if [[ ! -f miniconda/bin/activate ]]; then
+    echo No activate file;
+    exit 1;
+fi
+source "$PWD/miniconda/bin/activate"
+
+# install gfortran - which is not installed by limetr but required for it to work
+conda install -c conda-forge fortran-compiler
```

Comment:
per our conversation - the problem was not that conda wasn't activating correctly with the above call. the problem is that travis doesn't have gfortran installed by default, but the IHME cluster does.